### ISSUE TITLE
Fix highlight of a Kind in Cluster Dashboard

### ISSUE THREAD

**Issue by devcatalin** [kubeshop/monokle#3502]
In the Cluster Dashboard, a currently selected kind looks like this:

<img width="393" alt="image" src="https://user-images.githubusercontent.com/20538711/227222791-3e736c4f-88bb-42ee-992b-570b54608cf8.png">

if we hover it, it looks like this:

<img width="386" alt="image" src="https://user-images.githubusercontent.com/20538711/227222858-90adc60b-9525-46ae-ae9f-59db8266d997.png">

but, what we should do instead is what we do in the Explorer -> Navigator:

selected kind looks like this:
<img width="376" alt="image" src="https://user-images.githubusercontent.com/20538711/227223017-d9410d64-87c5-4d2e-877e-323526fd1816.png">

and if we hover it it looks like this:
<img width="398" alt="image" src="https://user-images.githubusercontent.com/20538711/227223073-07f9e0ad-6836-4165-9939-eede0d646ed1.png">

basically, the gradient remains the same but it's darkened
